setup.py

```diff
@@ -1,19 +1,57 @@
+import distutils
 import glob
 import os
 import setuptools
 import setuptools.command.build_ext
+import setuptools.command.install
 import setuptools.dist
 import sys
-import setuptools.command.install
 
-setuptools.dist.Distribution({
```

Comment:
It looked like dead code to me (could have been wrong). I think the code makes an instance of Distribution and then doesn't assign it to a variable, so the instance is garbage collected?